zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/service/ObjectManagementService.kt

```diff
@@ -0,0 +1,32 @@
+package com.ritense.objectmanagement.service
+
+import com.ritense.objectmanagement.domain.ObjectManagement
+import com.ritense.objectmanagement.repository.ObjectManagementRepository
+import java.util.UUID
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.http.HttpStatus
+import org.springframework.web.server.ResponseStatusException
+
+class ObjectManagementService(
+    private val objectManagementRepository: ObjectManagementRepository
+) {
+
+    fun createAndUpdate(objectManagement: ObjectManagement): ObjectManagement {
+        val databaseObjectManagement = objectManagementRepository.findByTitle(objectManagement.title)
+        if (databaseObjectManagement != null){
+            throw ResponseStatusException(
+                HttpStatus.CONFLICT,
+                "This title already exists please chose another title"
```

Comment:
When updating, Its normal that there's already an object with that title.

Maybe make a separate method?